FastSimulation/Tracking/test/testGeneralTracks.cc

```diff
@@ -122,31 +127,28 @@ void testGeneralTracks::bookHistograms(DQMStore::IBooker& ibooker,
 }
 
 testGeneralTracks::~testGeneralTracks() {
-  std::cout << "\t\t Number of Tracks " << std::endl;
-  std::cout << "\tFULL\t" << numfull << "\t HP= " << numfullHP << std::endl;
-  std::cout << "\tFAST\t" << numfast << "\t HP= " << numfastHP << std::endl;
+  edm::LogPrint("testGeneralTracks") << "\t\t Number of Tracks " << std::endl;
+  edm::LogPrint("testGeneralTracks") << "\tFULL\t" << numfull << "\t HP= " << numfullHP << std::endl;
+  edm::LogPrint("testGeneralTracks") << "\tFAST\t" << numfast << "\t HP= " << numfastHP << std::endl;
```

Comment:
```suggestion
  edm::LogPrint("testGeneralTracks") << "\t\t Number of Tracks ";
  edm::LogPrint("testGeneralTracks") << "\tFULL\t" << numfull << "\t HP= ";
  edm::LogPrint("testGeneralTracks") << "\tFAST\t" << numfast << "\t HP= ";
```
`endl` is not needed at the end of the LogMessages, but it also doesn't hurt: decide yourself what you like more